solutions/swb-reference/src/templates/onboard-account.cfn.yaml

```diff
@@ -387,7 +386,6 @@ Resources:
   HostingAccountHandlerRole:
     Type: 'AWS::IAM::Role'
     Properties:
-      RoleName: !Join ['-', [Ref: Namespace, 'hosting-account-role']]     # Confirm the suffix `hosting-account-role` matches with the suffix in SWBStack for AccountHandler
```

Comment:
Is getting rid of the rolenames fixing a warning?

Reply:
Did you make sure that there are not placeholders that expect this role name format as principals or resources in permissions? i.e. is there any instances of `${namespace}-hosting-account-role` in the code base? Taking this out would break that.

Reply:
Yeah, I think this format is expected some places: https://github.com/aws-solutions/solution-spark-on-aws/blob/main/solutions/swb-reference/src/SWBStack.ts#L557 and https://github.com/aws-solutions/solution-spark-on-aws/blob/main/solutions/swb-reference/src/SWBStack.ts#L650.

Have you verified these permissions still work after? If not, I would take out this fix in this PR as this is supposed to just do placeholder suppressions to triage later. In triaging later, you can verify this fix is okay.